client/src/ui/landingpage/LandingPageHero.tsx

```diff
@@ -0,0 +1,69 @@
+import Image from "next/image";
+//import { TextCarousel } from "@/ui/TextCarousel";
+import { Typography } from "@mui/material";
+import Link from "@mui/material/Link";
+
+export function LandingPageHero() {
+  return (
+    <div style={{
+      height: "90vh",
+      display: "flex",
+      alignItems: "center",
+      justifyContent: "center",
+    }}>
+      <div style={{
+            display: "flex",
+            flexDirection: "column",
+            width: "90%",
+            height: "90%",
+            backgroundColor: "white",
+            border: "10px solid #E0E4F2",
+            boxShadow: "#6d6d6d 0px 0px 10px",
+            borderRadius: "20px",
+            alignItems: "center",
+            justifyContent: "center",
+      }}>
+        <div style={{
+              display: "flex",
+              flexDirection: "row",
+              alignItems: "center", 
+              justifyContent: "space-between",
+              width: "80%",
+        }}>
+            <Typography variant="h4">
```

Comment:
I don't know if this is right, but it seems like the CarouselText is not aligned vertically the same way as the logo. The CarouselText seems to be aligned vertically higher and not centered like the SqueakPeek logo

![image](https://github.com/user-attachments/assets/64b17b77-4d75-4eb1-8596-bdb191547a1f)

Reply:
imaged needed to be cropped a little. Fixed